script/libtest.bash

```diff
@@ -438,7 +438,7 @@ create_http_trigger(){
     local func=${1:?}; shift
     local domain=${1-""};
     local subpath=${2-""};
-    local basicauth=${3-""};
+    local var3=${3-""};
```

Comment:
can use two different arguments for both? if you want to leave an empty parameter you can use `""` like:

```
create_http_trigger foo foo.com /foo "" cors
```

Reply:
will change it like this.